src/shared/components/oncoprint/controls/styles.scss

```diff
@@ -84,7 +87,7 @@
     }
 
     .Select-menu-outer {
-      width: 300px;
+      width: 100%;
```

Comment:
do you need this class at all?  possibly not.

Reply:
Nom its not needed. I removed this css entry.